README.md

```diff
@@ -20,6 +20,37 @@ on github. And you have to have pushed your commits to the pull request branch o
 - Will detect if you do not have a pull request for your branch. And give an error.
 - Will detect if the pull request branch on github does not have the same commit as your current branch. And give an error
 
+#### Merging pull request from forked repository
+##### 1. Checkout and pull master on local repository
+``` bash
+$ git checkout master
+$ git pull
+```
+
+##### 2. Checkout new branch on local repository
+``` bash
+$ git checkout -b merge-external-pull-request
+```
+
+##### 3. Cherry pick commits from pull request
+Copy the commit id from the pull request in GitHub.
+
+``` bash
+$ git cherry-pick -x {commit id}
+```
+
+Repeat as necessary.
+
+##### 4. Create a new pull request
+``` bash
+$ hub pull-request -m "merge external pull request"
+```
+
+##### 5. Merge new pull request
+``` bash
+$ tcmerge "merge-external-pull-request"
```

Comment:
Here you can do
```
tcmerge "merge-external-pull-request Fixes #[EXTERNAL_PULL_REQUEST_NUMBER]"
```
That way the external PR will be closed when merge to master happens, and that merge commit will be linked to the extrnal PR.